supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -576,9 +576,16 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     }
   }
 
-  private def handleSuccess(outputMappings: Try[JobOutputs], returnCode: Int, executionHandle: ExecutionHandle): ExecutionHandle = {
+  private def gatherJobOutputFiles: Map[String,String] = {
+      val callOutputFiles = List(jesCallPaths.stderrPath, jesCallPaths.returnCodePath, jesCallPaths.stdoutPath, gcsExecPath)
+      val mapOfFiles = callOutputFiles map { case(path) => path.getFileName.toString -> path.toString } toMap
+      val result = mapOfFiles ++ Map("callRootPath" -> callRootPath.toString)
```

Comment:
or perhaps this should be a separate parameter since it's not really a call output file but seems to be something that's just along for the ride